test/loguniform.jl

```diff
@@ -0,0 +1,39 @@
+module TestLogUniform
+using Test
+using Distributions
+import Random
+
+@testset "LogUniform" begin
```

Comment:
@devmotion how should I test LogUniform?

Reply:
Is it acceptable to calculate some values using `scipy.stats.loguniform` and hardcode these here as reference?

Reply:
We should test all definitions, i.e., methods and types defined in loguniform.jl. All methods should be type-stable, i.e., checked with `@inferred`. The correctness of e.g. `mean`, `var`, `pdf`, `logpdf`, `cdf`, and `quantile` can be checked with hard-coded values, e.g., obtained from `scipy`. Consistency checks are also simple and useful (e.g., `cdf(quantile(d, p)) \isapprox p` or similar).

Reply:
In particular, you can test `median`, `quantile` and `cdf` against the corresponding transformed median, quantile, cdf of the uniform distribution.